api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -1343,22 +1343,23 @@ public void retireOrderFrequency_shouldRetireGivenOrderFrequency() {
 	
 	@Test
 	public void unretireOrderFrequency_shouldUnretireGivenOrderFrequency() {
-		OrderFrequency orderFrequency = orderService.getOrderFrequency(3);
+		executeDataSet("org/openmrs/api/include/OrderServiceTest-otherOrderFrequencies.xml");
+		OrderFrequency orderFrequency = orderService.getOrderFrequency(103);
 		assertNotNull(orderFrequency);
 		assertTrue(orderFrequency.getRetired());
-		assertEquals("Some Retire Reason", orderFrequency.getRetireReason());
+		assertEquals(null,orderFrequency.getRetireReason());
```

Comment:
@jtatia could you explain a bit why you introduced the OrderServiceTest-otherOrderFrequencies.xml dataset in this test? Wouldn't you make the original test pass without this?